deploy/cluster-libchart/templates/_network.tpl

```diff
@@ -0,0 +1,14 @@
+{{/*
+Define component services
+*/}}
+{{- define "cluster-libchart.componentServices" }}
+services:
+  {{- if .Values.hostNetworkAccessible }}
+  - name: vpc
+    serviceType: NodePort
```

Comment:
"hostNetwork" is not exactly a NodePort behavior, I would suggest refactor key to `enableNodePort`

Reply:
`NodePort` is wrong, I modified it to LoadBalancer that is consistent with the `kbcli cluster expose` command.